### PR TITLE
Master mi

### PR DESCRIPTION
**Pull request by sahar-shafiee**
uyjgyjhkhkjkjk

### CODE REVIEW

**Comment by likecodingloveproblems** [comment]
i have some questions

**Comment by likecodingloveproblems** [comment]
this PR has conflict

File: .storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  "stories": [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
```
[the remark this answers is not in the document]
Comment (likecodingloveproblems) [reply .storybook/main.js:6]: this is pending @sahar-shafiee

**Review by likecodingloveproblems — CHANGES_REQUESTED** [review]
please send it to review in progress!

File: containers/creat/component/InputPicture/InputPicture.module.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
 .icon_change_pic {
-    cursor: pointer;
     width: 33px;
     height: 33px;
     background: rgb(255, 255, 255);
     backdrop-filter: blur(15px);
     border-radius: 10px;
     top: 10px;
     transition: all 0.3s ease;
+    cursor: pointer;
```
Comment (likecodingloveproblems) [inline containers/creat/component/InputPicture/InputPicture.module.scss:9]: this must be changed @sahar-shafiee

**Review by likecodingloveproblems — CHANGES_REQUESTED** [review]

File: pages/_app.jsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ function MyApp({ Component, pageProps }) {
 
   const REFRESH_TOKEN_TIMEOUT = 300000;
   const Layout = Component.Layout || MyLayout;
-
+  console.log("test");
```
Comment (likecodingloveproblems) [inline pages/_app.jsx:22]: `console.log('test');`

File: stories/Button.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (likecodingloveproblems) [inline stories/Button.stories.jsx:1]: Use next js instead of react
`import Next from 'next';`

**Review by likecodingloveproblems — CHANGES_REQUESTED** [review]

File: containers/creat/component/InputPicture/InputPicture.module.scss (hunk 2 above)
Comment (sahar-shafiee) [reply containers/creat/component/InputPicture/InputPicture.module.scss:9]: yrtyytytu

File: .storybook/main.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions"
```
Comment (likecodingloveproblems) [inline .storybook/main.js:9]:
```suggestion
console.log('yahhhh')
```

File: stories/Button.jsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+   */
+  onClick: PropTypes.func,
+};
+
+Button.defaultProps = {
+  backgroundColor: null,
+  primary: false,
+  size: 'medium',
+  onClick: undefined,
+};
```
Comment (likecodingloveproblems) [inline stories/Button.jsx:50]:
```suggestion
Button.defaultStats = {}

Comment (likecodingloveproblems) [inline stories/Button.jsx:50]:
```suggestion
Button.defaultStats = {}
```

**Pull request closed without merging**